src/dma/traits.rs

```diff
@@ -102,19 +102,17 @@ pub trait Stream: Sealed {
 /// Trait for Double-Buffered DMA streams
 pub trait DoubleBufferedStream: Stream + Sealed {
     /// Set the peripheral address (par) for the DMA stream.
-    unsafe fn set_peripheral_address(&mut self, value: u32);
+    unsafe fn set_peripheral_address(&mut self, value: usize);
 
     /// Set the memory address (m0ar) for the DMA stream.
```

Comment:
Technically this comment is out-of-date now.

"Set the memory address pointer for the DMA stream. Can be either m0ar or m1ar"